scripts/parse-tsdocs.ts

```diff
@@ -1,27 +1,9 @@
 /* eslint-disable no-console */
-import {
-  withCompilerOptions,
```

Comment:
is this intentional?

Reply:
Yeah, I had to move all the utils to a separate file so I could import them into the `.spec` file